packages/common/sfnStep.js

```diff
@@ -1,13 +1,36 @@
 'use strict';
 
-const { s3, pullStepFunctionEvent } = require('@cumulus/common/aws');
-const StepFunctions = require('@cumulus/common/StepFunctions');
+const isNil = require('lodash.isnil');
+
+const { pullStepFunctionEvent } = require('./aws');
+const log = require('./log');
+const StepFunctions = require('./StepFunctions');
 
 /**
  * `SfnStep` provides methods for getting the output of a step within an AWS
  * Step Function for a specific execution.
 */
 class SfnStep {
+  constructor() {
```

Comment:
This file has pretty low coverage. I know you moved a lot of it, but we should try to test at least the functions that you added.

Reply:
I'm seeing that some of these functions are leftovers that I don't need. Previously, this file was in the `integration-tests` package, so not counting against our coverage, but I needed it for my Lambda. Do you want me to add unit tests for more of the methods?